setup.cfg

```diff
@@ -60,3 +60,4 @@ ara.cli =
     host show = ara.cli.host:HostShow
     stats list = ara.cli.stats:StatsList
     stats show = ara.cli.stats:StatsShow
+    static generate = ara.cli.static:StaticGenerate
```

Comment:
I don't know, I guess I namespace the commands by default in expectation that there might be more commands relevant to static management. I don't have a strong opinion on this one.